ironfish-rust/src/transaction/mod.rs

```diff
@@ -698,7 +703,7 @@ fn value_balance_to_point(value: i64) -> Result<ExtendedPoint, IronfishError> {
         None => return Err(IronfishError::IllegalValue),
     };
 
-    let mut value_balance = VALUE_COMMITMENT_VALUE_GENERATOR * jubjub::Fr::from(abs);
+    let mut value_balance = NATIVE_ASSET_GENERATOR * jubjub::Fr::from(abs);
```

Comment:
This is the same value - but we'll want to remove any usage of `VALUE_COMMITMENT_VALUE_GENERATOR` eventually and instead only rely on `NATIVE_ASSET_GENERATOR` (which is just a rename of `VALUE_COMMITMENT_VALUE_GENERATOR`) when working with the native asset - otherwise generator points should be using the asset's specific generator